apps/trustlab/package.json

```diff
@@ -19,16 +19,17 @@
     "@commons-ui/payload": "workspace:*",
     "@mui/utils": "catalog:",
     "@next/third-parties": "catalog:",
-    "@payloadcms/db-sqlite": "catalog:payload-v3",
     "@payloadcms/db-mongodb": "catalog:payload-v3",
+    "@payloadcms/email-nodemailer": "catalog:payload-v3",
     "@payloadcms/next": "catalog:payload-v3",
     "@payloadcms/plugin-cloud-storage": "catalog:payload-v3",
-    "@payloadcms/richtext-lexical": "catalog:payload-v3",
-    "@payloadcms/storage-s3": "catalog:payload-v3",
     "@payloadcms/plugin-seo": "catalog:payload-v3",
     "@payloadcms/plugin-nested-docs": "catalog:payload-v3",
+    "@payloadcms/richtext-lexical": "catalog:payload-v3",
+    "@payloadcms/storage-s3": "catalog:payload-v3",
     "@payloadcms/ui": "catalog:payload-v3",
     "cross-env": "catalog:",
+    "graphql": "catalog:payload-v3",
```

Comment:
@kilemensi The reason for adding this was because we do not need graphql. If we do need the package, we should then disbale it in the config 
```
  graphQL:{
    disable: true,
  },
```

Reply:
Whether we use it or not @kelvinkipruto , Payload [**_expects_**](https://github.com/payloadcms/payload/blob/98283ca18c1e68c44249ee7c64367b61d98d49db/packages/next/package.json#L126-L127) that you'll install it.

Yes, disabling it (via a build time arg or env var) is probably the right approach. Lets add it to lower priority issues/tasks we need to do when we're about to go live.